ui/org.eclipse.pde.core/text/org/eclipse/pde/internal/core/text/plugin/PluginImportNode.java

```diff
@@ -20,6 +20,18 @@
 public class PluginImportNode extends PluginObjectNode implements IPluginImport {
 
 	private static final long serialVersionUID = 1L;
+	private String fFilter;
+
+	@Override
+	public void setFilter(String filter) {
+		this.fFilter = filter;
```

Comment:
As this represents the PluginImport node of an XML file and looking at the other methods, the implementation seems to have to look more like
```suggestion
		setXMLAttribute(P_FILTER, filter);
```
with a new constant `String P_FILTER = "filter"; //$NON-NLS-1$` added in `IPluginImport`.